Languages/fa.js

```diff
@@ -0,0 +1,400 @@
+﻿export const Dictionary = {
+    newVersionTitle: "به روز رسانی موجود است",
+    newVersion: "یک نسخه‌ی جدید یافت شد.",
+    innovations: "نوآوری‌ها",
+    installWhenClosing: "پس از بستن برنامه نصب کن",
+    installNowAndRestart: "هم‌اکنون نصب کن",
+    new: 'جدید',
+    load: 'بارگزاری',
+    cancel: 'لغو',
+    save: 'ذخیره',
+    saveAndClose: 'ذخیره و بستن',
+    options: 'تنظیمات',
+    ok: 'باشه',
+    create: 'بساز',
+    yes: 'آره',
+    no: 'نه',
+    unknown: 'ناشناس',
+    chosenOne: 'شخص اصلی',
+    importFile: 'فایل وارد کن',
+    exportFile: 'فایل خارج کن',
+    value: 'محتوا',
+    count: 'دفعات به کار رفته',
+    empty: 'هیچی انتخاب نشده',
+    optionModals: {
+        flags: {
+            title: 'پرچم‌های شخص',
+            name: 'نام',
+            color: 'رنگ'
+        }
+    },
+    routing: {
+        Chart: 'در حال آماده‌سازی نمای درختی...',
+        PersonDatagrid: 'در حال آماده‌سازی نمای جدولی...',
+        Export: 'در حال آماده‌سازی خروجی...',
+        Preview: 'در حال آماده‌سازی پیش‌نمایش...',
+        Families: 'در حال آماده‌سازی خانواده‌ها...',
+        Documents: 'در حال آماده‌سازی اسناد...',
+        Suggestions: 'در حال آماده‌سازی پیشنهادها...',
+    },
+    suggestions: {
+        cities: 'شهرها',
+        countries: 'کشورها',
+        citiesInCountry: 'شهر در کشور',
+        firstNames: 'نام کوچک',
+        lastNames: 'نام خانوادگی',
+        religions: 'مذهب',
+        education: 'تحصیلات',
+        jobs: 'شغل',
+        // TODO: Review
```

Comment:
@SepSol ... TODO? you? me? 😄

Reply:
That's for myself 😁 I didn't know where were some of the texts in the app, so I thought maybe I mark the ones that I'm not sure if are appropriate or not. I'll remove them once I make sure, or you can remove them now, I have my own copy! 😄

Reply:
ok 👍